frontend/src/packages/business/business-api.ts

```diff
@@ -0,0 +1,58 @@
+import { ApiPath, ContentType } from '~/libs/enums/enums.js';
+import { HttpApi } from '~/libs/packages/api/http-api.js';
+import { type IHttp } from '~/libs/packages/http/http.js';
+import { type IStorage } from '~/libs/packages/storage/storage.js';
+import {
+  type PaginationParameters,
+  type TruckEntity,
+} from '~/libs/types/types.js';
+
+import { type TruckAddRequestDto } from '../trucks/libs/types/types.js';
+import { BusinessApiPath } from './libs/enums/enums.js';
+
+type Constructor = {
+  baseUrl: string;
+  http: IHttp;
+  storage: IStorage;
+};
+
+class BusinessApi extends HttpApi {
+  public constructor({ baseUrl, http, storage }: Constructor) {
+    super({ path: ApiPath.BUSINESS, baseUrl, http, storage });
+  }
+
+  public async findAllTrucksByBusinessId(
+    query?: PaginationParameters,
+  ): Promise<{ items: TruckEntity[]; total: number }> {
+    const queryParameters = query
+      ? `?pageIndex=${query.pageIndex}&pageSize=${query.pageSize}`
+      : '';
+
```

Comment:
actually there will be no need to find all trucks without pagination, so query should not be optional

Reply:
If I remember correctly, Denis's task **(th-214)** was to find all the trucks for the business in one request. And he was using this endpoint. And so it's not a problem to fix it if it's really not needed

Reply:
I don't think we should build query string manually